File: src/Hoops.Core/Models/WebContentType.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Hoops.Core.Interface;

namespace Hoops.Core.Models;

[Table("WebContentType")]
public class WebContentType : IAuditable
{
    [Key]
    public int WebContentTypeId { get; set; }
    public string WebContentTypeDescription { get; set; }

    public DateTime? CreatedDate { get; set; }
    public int? CreatedUser { get; set; }
    public DateTime? ModifiedDate { get; set; }
    public int? ModifiedUser { get; set; }
}

File: hoops.ui/src/domain/webContentType.ts
export class WebContentType {
    public webContentTypeId!: number;
    public webContentTypeDescription!: string;
    public createdDate?: Date | string | null;
    public createdUser?: number | null;
    public modifiedDate?: Date | string | null;
    public modifiedUser?: number | null;
    constructor() {}
}
